### DISCUSSION TITLE
Failed Compile of 3X3 Manual Mesh

### DISCUSSION THREAD

**Discussion by Cal-LF** [Jyers/Marlin#620] [Q&A] [answered]
Tried to compile twice but got a fail.

![Compile](https://user-images.githubusercontent.com/76499230/116199636-5785e580-a705-11eb-89d6-a2c699229d38.jpg)

**Comment by smooreace** [comment]
What did you edit?

**Comment by duncio** [comment]
Same here, same error message

**Comment by smooreace** [comment]
Again, what did you edit?

**Reply by duncio** [reply]
I copied `Configuration.h` and `Configuration_adv.h` from ManualMesh-3x3 E3V2 Templates folder, didn't make any changes to it.

**Comment by Jyers** [comment, marked as answer]
Sorry, too many conditionals to keep track of and programming way too late at night.... I'll fix this now

**Reply by Jyers** [reply]
Alright, should be good now

**Reply by duncio** [reply]
Thank you! Working great now!
